data.php

```diff
@@ -24,7 +24,7 @@
         exit();
     }
 
-    if ((strlen($_POST["local_brewery"]) >= 4) OR ($_POST["local_brewery"] == "")){
+    if ((strlen($_POST["local_brewery"]) >= 4) or ($_POST["local_brewery"] == "")) {
```

Comment:
You don't need to change this, at this point it doesn't matter, but PSR-12 says to use `||` over `or` and `&&` over `and`. I personally strongly dislike their decision, `and` and `or` are so much more readable, but nevermind!

Just an FYI but don't worry about changing it

Reply:
good to know